tests/components/upnp/test_sensor.py

```diff
@@ -74,13 +74,9 @@ async def test_upnp_sensors(hass: HomeAssistant, setup_integration: MockConfigEn
     assert wan_status_state.state == "Disconnected"
 
 
-async def test_derived_upnp_sensors(
-    hass: HomeAssistant, setup_integration: MockConfigEntry
-):
+async def test_derived_upnp_sensors(hass: HomeAssistant, config_entry: MockConfigEntry):
     """Test derived sensors."""
-    coordinator: UpnpDataUpdateCoordinator = hass.data[DOMAIN][
-        setup_integration.entry_id
-    ]
+    coordinator: UpnpDataUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id]
```

Comment:
Why shouldn't access integration details like hass.data or the coordinator in tests.